sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py

```diff
@@ -8,7 +8,10 @@
 class PortChangeEvent:
     PORT_ADD = 0
     PORT_REMOVE = 1
-    def __init__(self, port_name, port_index, asic_id, event_type):
+    PORT_SET = 2
+    PORT_DEL = 3
```

Comment:
it hard to differentiate the four actions here from the name, would you mind rephrasing?

Reply:
Sure, and do you have any suggestions?

Reply:
While I could surely update these namings, but I don't know which one is better, could you please help me on this?